.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ templates:
       ignore: /.*/
 
 
-var_1: &docker_image circleci/node:12.18.0-browsers
+var_1: &docker_image circleci/node:14.15.4-browsers
```

Comment:
Nitpick: `cimg/node:lts-browsers` always point to latest NodeJS LTS, so you don't need to think about it anymore.

Reply:
I used an exact version, because CircleCI recommends to do so in their docs for having more reliability. But actually, not having the LTS version brought me to a problem here, so I will give it a try.